src/mumble/XMLTools.h

```diff
@@ -0,0 +1,42 @@
+#ifndef MUMBLE_MUMBLE_XMLTOOLS_H_
+#define MUMBLE_MUMBLE_XMLTOOLS_H_
+
+#include <QObject>
+#include <QMap>
+
+class QXmlStreamReader;
+class QXmlStreamWriter;
+
+class XMLTools : public QObject

```

Comment:
The class is not coding-style compliant.

- Wrong cruly placement.
- We tend to indent public, private, protected, (etc.) keywords a level.
- We tend to indent Q_OBJECT two levels.
- It uses tabs instead of spaces for alignment. (for recurseParse)
- It has empty signals and public slots, just drop them.